app/src/main/java/com/fluffybros/tujefe/ui/add/ManualAddFragment.kt

```diff
@@ -0,0 +1,14 @@
+package com.fluffybros.tujefe.ui.add
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import com.fluffybros.tujefe.R
+
+class ManualAddFragment : Fragment(R.layout.fragment_manual_add) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
you don't need this function if all you're doing is calling super.onViewCreated - remove it